src/std/sys.c

```diff
@@ -130,7 +136,11 @@ HL_PRIM vbyte *hl_sys_locale() {
 
 HL_PRIM void hl_sys_print( vbyte *msg ) {
 	hl_blocking(true);
+#ifdef HL_ANDROID
+	LOG_ANDROID(hl_to_utf8(msg));
+#else

```

Comment:
maybe nitpicking, but would it not be the best to handle this directly in uprintf ?

Reply:
We need to use __android_log_print to get standard output through adb.

Reply:
Sure, but I was thinking that could be handled at a slightly lower level (uprintf implementation) instead of hl_sys_print

Reply:
OK, no problem moving that in src/std/ucs2.c uprintf implementation.